tests/testthat/test-as_factor.R

```diff
@@ -16,3 +16,10 @@ test_that("supports NA (#89)", {
   x <- c("a", "z", "g", NA)
   expect_equal(as_factor(x), fct_inorder(x))
 })
+
+test_that("converts ordered factor into plain factor (#216)", {
+  x <- factor(c("a", "z", "g"), ordered = TRUE)
+  x <- as_factor(x)
+  expect_false(is.ordered(x))
+  expect_equal(attributes(x)$class, "factor")
```

Comment:
I think I'd just do `expect_s3_class(as_factor(x), "factor", exact = "TRUE")` (i.e. you only need one test)

Reply:
done